python/tests/integration/test_methods.py

```diff
@@ -296,7 +296,7 @@ def test_search_look_and_run(sdk: mtds.LookerSDK):
     """run_look() should return CSV and JSON
     CSV will use column descriptions
     JSON will use column names
-    JSON_LABEL will use column descriptions
+    JSON_DETAIL will use column descriptions
```

Comment:
Did the behavior of JSON_LABEL change? Wasn't this test intended to verify the behavior of the new format, JSON_LABEL?

Reply:
You're right. I just reverted this change.